ocfweb/docs/docs/services/vhost.md

```diff
@@ -135,9 +135,17 @@ does not process nor advise off-site hosting requests, which can be
 [submitted directly to the university][offsite] (however, feel free to keep
 us informed if you have an existing OCF account).
 
-In other words, to comply with university policy, the OCF may only provide
+   In other words, to comply with university policy, the OCF may only provide
```

Comment:
It's not indented in the markdown, but when rendered, the bullet list gets indented because of the list. Adding the space forces the paragraph to be indented when rendered.

![screenshot_2017-04-13_23-02-52](https://cloud.githubusercontent.com/assets/1916822/25034730/730c56ec-209d-11e7-933a-4ae57b5b52af.png)

vs

![screenshot_2017-04-13_23-07-26](https://cloud.githubusercontent.com/assets/1916822/25034796/038aac5a-209e-11e7-8d5c-01ce8d5bc0e2.png)

without the indentation, that paragraph renders unindented and it breaks the continuity of the bullet list.